README.md

```diff
@@ -4,6 +4,35 @@ OpenShift deployment code based on Tekton.
 
 DISCLAIMER: This is just the inital setup, and as such is very basic at the moment.
 
+## BuildConfigs and ImageStreams
+
+### buildconfig_imagestream_templates.yaml
+
+A generic template designed to allow the creation of a BuildConfig and two ImageStreams, one for the base container for the build and one for the output. 
+
+```
+BUILDCONFIG_NAME - the name to give the BuildConfig
+OUTPUT_IMAGESTREAM_NAME - the name to give the ImageStream that will contain your output image
+BASE_IMAGESTREAM_NAME - the base image to use with your Dockerfile
+GIT_SOURCE - the GIT repo you'll pull the Dockerfile from
+GIT_BRANCH - the branch of the repo to use
+CONTEXT_DIR - the directory containing your Dockerfile in the Git repo
+SOURCE_IMAGE_FOR_BASE - the url to access the source image you wish to use as your base (this defaults to the Red Hat RHEL8 UBI minimal image)
+```
```

Comment:
Would be good to add the other three possible paremeters here, NAMESPACE, OUTPUT_IMAGE_STREAM_TAG and BASE_IMAGESTREAM_TAG, even if they have defaults.

Reply:
Updated and removed the unused namespace variable